core/src/test/java/org/elasticsearch/action/delete/DeleteResponseTests.java

```diff
@@ -103,11 +103,11 @@ public void testToAndFromXContent() throws IOException {
 
         Tuple<ReplicationResponse.ShardInfo, ReplicationResponse.ShardInfo> shardInfos = RandomObjects.randomShardInfo(random());
 
-        DeleteResponse actual = new DeleteResponse(new ShardId(index, indexUUid, shardId), type, id, seqNo, version, found);
+        DeleteResponse actual = new DeleteResponse(new ShardId(index, indexUUid, shardId), type, id, seqNo, version, found, 0L);
```

Comment:
You should use a random took here. Something like `TimeValue took = randomTimeValue(0, Long.MAX_VALUE)`.